src/platform/CYW30739/KeyValueStoreManagerImpl.cpp

```diff
@@ -75,14 +74,24 @@ CHIP_ERROR KeyValueStoreManagerImpl::_Get(const char * key, void * value, size_t
 
     VerifyOrReturnError(offset_bytes == 0, CHIP_ERROR_NOT_IMPLEMENTED);
 
-    const size_t keyLength = strnlen(key, CHIP_CONFIG_PERSISTED_STORAGE_MAX_KEY_LENGTH);
-    VerifyOrExit(keyLength != 0 && keyLength <= CHIP_CONFIG_PERSISTED_STORAGE_MAX_KEY_LENGTH &&
+    const size_t keyLength = strnlen(key, PersistentStorageDelegate::kKeyLengthMax);
+    VerifyOrExit(keyLength != 0 && keyLength <= PersistentStorageDelegate::kKeyLengthMax &&
                      value_size <= kMaxPersistedValueLengthSupported,
                  err = CHIP_ERROR_INVALID_ARGUMENT);
 
     entry = FindEntry(key);
     VerifyOrExit(entry != nullptr, err = CHIP_ERROR_PERSISTED_STORAGE_VALUE_NOT_FOUND);
-    VerifyOrExit(value_size != 0, err = CHIP_ERROR_BUFFER_TOO_SMALL);
+
+    if (value_size == 0 || entry->GetValueSize() == 0)
+    {
+        if (read_bytes_size != nullptr)
+            *read_bytes_size = 0;
+
+        if (value_size >= entry->GetValueSize())
+            ExitNow(err = CHIP_NO_ERROR);
+        else
+            ExitNow(err = CHIP_ERROR_BUFFER_TOO_SMALL);
+    }
```

Comment:
Can you please add issues in https://github.com/orgs/project-chip/projects/3 about your platform's audit and if you had issues, please raise small issues attached to that project and mark with `fixes #XXX` in the PR description. I helps understand the impact of the audit. No need to changer the lines here, just wanted to make sure this comment was seen.

Reply:
It seems I don't have access to https://github.com/orgs/project-chip/projects/3. It gave me a 404 error.
How do I apply for access to the project?

Reply:
I have created an issue for this https://github.com/project-chip/connectedhomeip/issues/22030.